### ISSUE TITLE
Can't boot to install menu.

### ISSUE THREAD

**Issue by winkyzero** [huukhai/hackintosh-rog-b550i#7]
Hi, I have the almost same system like you, just has a different RAM (16GB) and VGA RX570. After run a few second then the screen become like Color tearing and stuck. Do I need to change the config.plist?
Thanks

**Comment by huukhai** [comment]
Yes, you need to change platform info and change the bios configuration (upgrade to the latest version of BIOS and enable 4G decoding).
https://dortania.github.io/OpenCore-Install-Guide/AMD/zen.html#platforminfo
https://dortania.github.io/OpenCore-Install-Guide/AMD/zen.html#amd-bios-settings

**Comment by winkyzero** [comment]
Thanks bro. I can boot right now. Just also need to enable Above 4G decoding on CMOS.
Thanks again.
